### ISSUE TITLE
Support for yaourt's -C?

### ISSUE THREAD

**Issue by wilbowma** [kitsunyan/pakku#23]
I'm trying to switch from `yaourt`, but I miss the `-C` feature that lets me quickly clean up any `*.pacnew`, etc, files.

**Comment by kitsunyan** [comment]
Actually I even thought about createing a merger tool for `*.pacnew` files. I think I'll do something like that.

**Comment by wilbowma** [comment]
Awesome, thanks! What did you have in mind for a merging tool? Launching, eg, vimdiff seems like the best case; I doubt things can be automatically merged well in general.

--

**Comment by kitsunyan** [comment]
Yes, it won't be an auto merger. I'm thinking about implementing it via `git add -p`, so user could inspect each diff and edit it. I found this thing quite convenient when I worked with git.

**Comment by wilbowma** [comment]
Oh that's a very good idea! Look forward to it.

--

**Comment by seafox** [comment]
@wilbowma for now you could try `pacui` [package](https://github.com/excalibur1234/pacui) with Maintain System option. It currently has no ability to perform selective editing on diff, just rewrite or clean up new config files (an external prog like meld might be invoked when new/backup of config file found, but no integration AFAIK exists yet).

**Comment by mrvik** [comment]
What about `pacdiff` from `community/pacman-contrib`?
It's a good tool to take care of the `pac{new,save}` files, but a implementation in `pakku` would be great.
